blackjack/blackjack/urls.py

```diff
@@ -22,6 +22,6 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
 
-    path("api/game", views.deal, ),
+    path("api/game", views.Game.deal(), ),
```

Comment:
this is good intuition (deal as a method of the game class), but see above

Reply:
fixed!